src/char_mapping.rs

```diff
@@ -59,7 +129,7 @@ table! {
             tape.take_given(this.segment_count())
         },
 
-        reserved_pad (u16), // reservedPad
+        reserved_pad (u16) = { 0 }, // reservedPad
```

Comment:
I realize that it’s my code, but let’s call it `reserved` to be consistent about how reserved fields are named.